best-practices.md

```diff
@@ -397,3 +398,15 @@ basic geographic filtering from listeners.
 The dynamic STAC API would then listen to the notifications and update its internal datastore whenever new data comes into
 the static catalog. Implementors have had success using AWS Lambda to do a full 'serverless' updating of the elasticsearch
 database, but it could just as easily be a server-based process.
+
+## Provenance
+
+A full provenance model is far beyond the scope of STAC, and the goal is to align with any good independent spec 
+that comes along for that. The [`derived_from` relation type in links](item-spec/item-spec.md#relation-types) is seen as
+a way to encourage fuller specs and at least start a linking structure that can be used as a jumping off point for more
+experiments in provenance tracking.
+
+Providers can provide [lineage](https://github.com/radiantearth/stac-spec/issues/863) information in the `providers` property,
```

Comment:
A little odd that we're linking to an issue here. But I hope the intent is we get to an extension in the spec before too long?

Reply:
The issue has some background on lineage, like that we may add sci:lineage and other ways you could add those info. Wanted to link that somewhere, but maybe not the best place? Happy to adopt any suggestion.

Reply:
I'd probably just summarize the key info from that ticket here...

Reply:
But I wouldn't consider all that best practices though...

Reply:
```suggestion
Providers can provide lineage information (see [this issue](https://github.com/radiantearth/stac-spec/issues/863) for background) in the `providers` property,
```

Reply:
Ok, the above suggestion would be a bit better for me - as a user I'd expect a link in a best practice doc to link me to some authoritative definition, not to a discussion.